client/src/components/PageFooter.vue

```diff
@@ -39,6 +39,7 @@ export default defineComponent({
 
 <style scoped>
 .landing-page-footer {
```

Comment:
can you remove 'landing' from the classnames here too

Reply:
Nice catch